### PR TITLE
Finishing #6252

### PR DESCRIPTION
**Pull request by weaverryan**
WIP - finishing #6252

Re-opening so I can tweak and review. Rebased to 2.8

### CODE REVIEW

File: book/from_flat_php_to_symfony2.rst [hunk 1]
```diff
@@ -559,10 +583,15 @@ them for you. Here's the same sample application, now built in Symfony::
         }
     }
 
+First we have a "controller class" which is a convenient way to group several
+"controllers" together. So methods inside a controller class are controllers
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: This phrase looks confusing:

```
So methods inside a controller class are controllers also called *actions*.
```

What about:

```
So methods inside a controller class are also controllers, but they are usually called *actions*.
```

File: book/from_flat_php_to_symfony2.rst [hunk 2]
```diff
@@ -559,10 +583,15 @@ them for you. Here's the same sample application, now built in Symfony::
...
+also called *actions*. They hold code which creates and returns the appropriate
+``Response`` object.
+
 The two controllers are still lightweight. Each uses the
 :doc:`Doctrine ORM library </book/doctrine>` to retrieve objects from the
 database and the Templating component to render a template and return a
-``Response`` object. The list template is now quite a bit simpler:
+``Response`` object. The list ``list.php`` template is now quite a bit simpler:
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: `The list list.php template ...` -> `The list.php template ...`

File: book/from_flat_php_to_symfony2.rst [hunk 3]
```diff
@@ -585,7 +614,7 @@ database and the Templating component to render a template and return a
         <?php endforeach ?>
     </ul>
 
-The layout is nearly identical:
+The layout ``layout.php`` is nearly identical:
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: `The layout layout.php is...` -> `The layout.php is...`

File: book/from_flat_php_to_symfony2.rst [hunk 4]
```diff
@@ -605,12 +634,13 @@ The layout is nearly identical:
 
 .. note::
 
-    The show template is left as an exercise, as it should be trivial to
-    create based on the list template.
+    The show ``show.php`` template is left as an exercise, as it should be trivial to
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: `show show.php template` -> `show.php template`

File: book/from_flat_php_to_symfony2.rst [hunk 5]
```diff
@@ -605,12 +634,13 @@ The layout is nearly identical:
...
+    create based on the list ``list.php`` template.
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: `list list.php template` -> `list.php template`

File: book/from_flat_php_to_symfony2.rst [hunk 6]
```diff
@@ -605,12 +634,13 @@ The layout is nearly identical:
...
 
-When Symfony's engine (called the ``Kernel``) boots up, it needs a map so
+When Symfony's engine (called the :term:`Kernel`) boots up, it needs a map so
```
Comment (javiereguiluz) [inline book/from_flat_php_to_symfony2.rst]: We've removed the glossary from Symfony Docs, so we can stop using `:term:`

File: book/http_fundamentals.rst [hunk 7]
```diff
@@ -69,13 +69,14 @@ In HTTP-speak, this HTTP request would actually look something like this:
...
+most important and contains three things: the HTTP method, the requested URI
+and HTTP protocol version being used.
 
 The URI (e.g. ``/``, ``/contact``, etc) is the unique address or location
 that identifies the resource the client wants. The HTTP method (e.g. ``GET``)
-defines what you want to *do* with the resource. The HTTP methods are the
-*verbs* of the request and define the few common ways that you can act upon
-the resource:
+defines what the client wants to *do* with the resource. The HTTP methods (also
+known as verbs) define the few common ways that the client can acts upon the
```
Comment (javiereguiluz) [inline book/http_fundamentals.rst]: `can acts upon` -> `can act upon`

File: book/http_fundamentals.rst [hunk 8]
```diff
@@ -246,61 +252,97 @@ have all the request information at your fingertips::
...
+application to use an object-oriented interface to construct the response that
+needs to be returned to the client::
 
     use Symfony\Component\HttpFoundation\Response;
 
     $response = new Response();
 
     $response->setContent('<html><body><h1>Hello world!</h1></body></html>');
-    $response->setStatusCode(Response::HTTP_OK);
+    $response->setStatusCode(Response::HTTP_OK);;
```
Comment (javiereguiluz) [inline book/http_fundamentals.rst]: `;;` -> `;`

File: book/installation.rst [hunk 9]
```diff
@@ -60,7 +60,7 @@ Creating the Symfony Application
 --------------------------------
 
 Once the Symfony Installer is available, create your first Symfony application
-with the ``new`` command:
+with the ``new`` command::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 10]
```diff
@@ -109,7 +109,7 @@ optional second argument of the ``new`` command:
     $ symfony new my_project 2.7.0-RC1
 
 The installer also supports a special version called ``lts`` which installs the
-most recent :ref:`Symfony LTS version <releases-lts>` available:
+most recent :ref:`Symfony LTS version <releases-lts>` available::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 11]
```diff
@@ -140,15 +140,15 @@ Start with :doc:`installing Composer globally </cookbook/composer>`.
 Creating a Symfony Application with Composer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once Composer is installed on your computer, execute the ``create-project``
-command to create a new Symfony application based on its latest stable version:
+Once Composer is installed on your computer, execute the ``create-project`` Composer
+command to create a new Symfony application based on its latest stable version::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 12]
```diff
@@ -140,15 +140,15 @@ Start with :doc:`installing Composer globally </cookbook/composer>`.
...
 
 .. code-block:: bash
 
     $ composer create-project symfony/framework-standard-edition my_project_name
 
 If you need to base your application on a specific Symfony version, provide that
-version as the second argument of the ``create-project`` command:
+version as the second argument of the ``create-project`` Composer command::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 13]
```diff
@@ -160,12 +160,14 @@ version as the second argument of the ``create-project`` command:
...
 
+    $ composer create-project -vvv symfony/framework-standard-edition my_project_name "2.3.*"
+
 Running the Symfony Application
 -------------------------------
 
 Symfony leverages the internal web server provided by PHP to run applications
 while developing them. Therefore, running a Symfony application is a matter of
-browsing the project directory and executing this command:
+browsing the project directory and executing this command::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 14]
```diff
@@ -183,28 +185,36 @@ Instead of the Welcome Page, you may see a blank page or an error page.
...
-The ``server:run`` command is only suitable while developing the application. In
-order to run Symfony applications on production servers, you'll have to configure
-your `Apache`_ or `Nginx`_ web server as explained in
-:doc:`/cookbook/configuration/web_server_configuration`.
+    PHP's internal web server is available in PHP 5.4 or higher versions. So, if you
+    still use the legacy PHP 5.3 version, you'll also have to configure `Apache`_ or
+    `Nginx`_ web server and virtual host.
 
 When you are finished working on your Symfony application, you can stop the
+<<<<<<< 2dc8c63868d4d5677c0ec3350d6bdac807452b2f
```
Comment (javiereguiluz) [inline book/installation.rst]: This is a merge leftover :)

File: book/installation.rst [hunk 15]
```diff
@@ -183,28 +185,36 @@ Instead of the Welcome Page, you may see a blank page or an error page.
...
+    $ php app/console server:stop
+>>>>>>> made some new changes, corrected mistakes, undo some foolish deletions
 
 Checking Symfony Application Configuration and Setup
 ----------------------------------------------------
 
 Symfony applications come with a visual server configuration tester to show if
 your environment is ready to use Symfony. Access the following URL to check your
-configuration:
+configuration::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 16]
```diff
@@ -299,7 +309,7 @@ and they are managed exclusively by Composer.
 
 Updating those third-party libraries frequently is a good practice to prevent bugs
 and security vulnerabilities. Execute the ``update`` Composer command to update
-them all at once:
+them all at once::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/installation.rst [hunk 17]
```diff
@@ -309,6 +319,7 @@ them all at once:
 Depending on the complexity of your project, this update process can take up to
 several minutes to complete.
 
+<<<<<<< 2dc8c63868d4d5677c0ec3350d6bdac807452b2f
```
Comment (javiereguiluz) [inline book/installation.rst]: A merge leftover :)

File: book/installation.rst [hunk 18]
```diff
@@ -330,7 +375,7 @@ conceived as a learning tool for Symfony newcomers and its source code contains
 tons of comments and helpful notes.
 
 In order to download the Symfony Demo application, execute the ``demo`` command
-of the Symfony Installer anywhere in your system:
+of the Symfony Installer anywhere in your system::
```
Comment (javiereguiluz) [inline book/installation.rst]: `::` -> `:`

File: book/page_creation.rst [hunk 19]
```diff
@@ -150,8 +206,8 @@ Dynamic URL Patterns: /lucky/number/{count}
 
 Woh, you're doing great! But Symfony's routing can do a lot more. Suppose
 now that you want a user to be able to go to ``/lucky/number/5`` to generate
-*5* lucky numbers at once. Update the route to have a ``{wildcard}`` part
-at the end:
+*5* lucky numbers at once. Update the route to have a "wildcard" placeholders
+``{counter}`` at the end::
```
Comment (javiereguiluz) [inline book/page_creation.rst]: `::` -> `:`

File: book/page_creation.rst [hunk 20]
```diff
@@ -337,30 +394,25 @@ also get a lot of shortcut methods, like ``render()``::
...
 ~~~~~~~~~~~~~~~~~~~
 
-If you refresh now, you'll get an error:
+If you refresh your browser now, you'll get an error:
 
     Unable to find template "lucky/number.html.twig"
 
 Fix that by creating a new ``app/Resources/views/lucky`` directory and putting
-a ``number.html.twig`` file inside of it:
+a ``number.html.twig`` file inside of it::
```
Comment (javiereguiluz) [inline book/page_creation.rst]: `::` -> `:`

File: book/page_creation.rst [hunk 21]
```diff
@@ -429,36 +484,39 @@ The ``app/`` directory also holds a few other things, like the cache directory
...
 
 Application Configuration
 -------------------------
 
-Symfony comes with several built-in bundles (open your ``app/AppKernel.php``
-file) and you'll probably install more. The main configuration file for bundles
-is ``app/config/config.yml``:
+Symfony Standard Edition comes with several built-in bundles (open your
+``app/AppKernel.php`` file) and you'll probably install more. The main configuration
+file for bundles is ``app/config/config.yml``::
```
Comment (javiereguiluz) [inline book/page_creation.rst]: `::` -> `:`

**Comment by javiereguiluz** [comment]
👍 

Status: reviewed

**Comment by WouterJ** [comment]
I love the irony of the "minor" tag when merging such big PRs ;) Btw, any reason you choose 2.8 and not 2.7?

**Comment by weaverryan** [comment]
@WouterJ oh damn - thanks for noticing that. I was thinking "2.8 is LTS" (which is true, but 2.7 is still supported). I'll patch it back

**Pull request merged**